swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -58,18 +58,21 @@ Follow the instructions given in this section to develop the service.
     The generated record type will be as follows.
 
     ```ballerina
-    type Doctor record {|
+    type Doctor record {
         string name;
         string hospital;
         string category;
         string availability;
         decimal fee;
-    |};
+    };
     ```
 
     > **Note:**
     > While it is possible to work with the JSON payload directly, using record types offers several advantages including enhanced type safety, data validation, and better tooling experience (e.g., completion).
 
+    > **Note:**
+    > When the JSON object fields are intended to precisely match the sample payload, the generated records can be updated to be [closed records](https://ballerina.io/learn/by-example/controlling-openness/), which would indicate that no other fields are allowed or expected.
```

Comment:
I don't think " the JSON object fields are intended" reads well. We expect the fields of the expected payload to match the sample payload.

Any issue with the suggestion in https://github.com/ballerina-platform/ballerina-dev-website/pull/8180#discussion_r1381150777?

Reply:
Ah no. Just wanted to improve